tests/network/localnet/liblocalnet.py

```diff
@@ -1,15 +1,39 @@
 from libs.net import netattachdef
-from libs.net.vmspec import add_network_interface, add_volume_disk
+from libs.net.traffic_generator import Client, Server
+from libs.net.vmspec import IP_ADDRESS, add_network_interface, add_volume_disk, lookup_iface_status
 from libs.vm.affinity import new_pod_anti_affinity
 from libs.vm.factory import base_vmspec, fedora_vm
 from libs.vm.spec import CloudInitNoCloud, Interface, Metadata, Multus, Network
 from libs.vm.vm import BaseVirtualMachine, cloudinitdisk_storage
 from tests.network.libs import cloudinit
 
-NETWORK_NAME = "localnet-network"
+_IPERF_SERVER_PORT = 5201
 
 
-def localnet_vm(namespace: str, name: str, network: str, cidr: str) -> BaseVirtualMachine:
+def run_vms(
+    *vms: BaseVirtualMachine,
+) -> tuple[BaseVirtualMachine, ...]:
+    for vm in vms:
+        vm.start()
+    for vm in vms:
+        vm.wait_for_ready_status(status=True)
+        vm.wait_for_agent_connected()
+    return vms
+
+
+def create_traffic_server(vm: BaseVirtualMachine) -> Server:
+    return Server(vm=vm, port=_IPERF_SERVER_PORT)
+
+
+def create_traffic_client(vm_server: BaseVirtualMachine, vm_client: BaseVirtualMachine, network_name: str) -> Client:
+    return Client(
+        vm=vm_client,
+        server_ip=lookup_iface_status(vm=vm_server, iface_name=network_name)[IP_ADDRESS],
+        server_port=_IPERF_SERVER_PORT,
+    )
+
+
+def localnet_vm(namespace: str, name: str, network: str, cidr: str, network_name: str) -> BaseVirtualMachine:
```

Comment:
Please change existing code in a separate commit so it is easy to review. I wont insist on this specific additional argument now, but it is not convenient for me to review the PR like this.

- Maybe it is only me, but as a caller to this function, how do I know what is the difference between `network` and `network_name`? You need a better name, possibly for both in order to be clear.
- You have not updated the docstring.

Reply:
https://github.com/RedHatQE/openshift-virtualization-tests/pull/900/commits/ff722413a0f0237cdc255d32274a89f4945bf418